src/Form.js

```diff
@@ -85,6 +85,22 @@ class Form {
         return data;
     }
 
+    /**
+     * Fetch specific data for the form.
+     *
+     * @param {array} fields
+     * @return {object}
+     */
+    only(fields) {
+        const data = {};
+
+        fields.forEach(property => {
```

Comment:
can this be implemented using the `filter` function? 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter

Then we can get rid of the temporary `data` variable.

Reply:
I used `reduce` instead so that it still returns an object, `filter` returns an array. See latest commit.